package.json

```diff
@@ -1,21 +1,57 @@
 {
-  "name": "jira-comment-visibility",
-  "version": "0.1.0",
-  "description": "Acts as a default comment visibility setting for Jira. Only works for our company internal instance for now.",
-  "scripts": {
-    "lint": "npx eslint .",
-    "fix": "npx eslint . --fix",
-    "build": "7z a dist/$(git symbolic-ref -q --short HEAD || git describe --tags --exact-match).zip . -xr@.buildignore"
-  },
-  "author": "Bastian Böhne",
-  "license": "Beer-Ware",
-  "devDependencies": {
-    "@html-eslint/eslint-plugin": "^0.14.1",
-    "@html-eslint/parser": "^0.14.0",
-    "eslint": "^8.24.0",
-    "eslint-config-standard": "^17.0.0",
-    "eslint-plugin-import": "^2.26.0",
-    "eslint-plugin-n": "^15.3.0",
-    "eslint-plugin-promise": "^6.0.1"
-  }
+	"private": true,
+	"scripts": {
+	  	"build:chrome": "parcel build source/mv3/manifest.json --no-content-hash --no-source-maps --dist-dir build/chrome --no-cache --detailed-report 0",
+	  	"build:firefox": "parcel build source/mv2/manifest.json --no-content-hash --no-source-maps --dist-dir build/firefox --no-cache --detailed-report 0",
+		"lint": "run-p lint:*",
+		"lint-fix": "run-p 'lint:* -- --fix'",
+		"lint:css": "stylelint source/**/*.css",
+		"lint:js": "xo",
+		"test": "run-p lint:* build:*",
+	  	"clean:chrome": "rm -rf build/chrome",
+	  	"clean:firefox": "rm -rf build/firefox",
+	  	"package:firefox": "web-ext build --overwrite-dest --source-dir build/firefox --filename firefox.zip",
+	  	"package:chrome": "web-ext build --overwrite-dest --source-dir build/chrome --filename chrome.zip",
+	  	"prepublish": "run-s clean:* build:* package:*",
+	  	"watch:chrome": "parcel watch source/mv3/manifest.json --dist-dir build/chrome --no-cache --no-hmr",
+	  	"watch:firefox": "parcel watch source/mv2/manifest.json --dist-dir build/firefox --no-cache"
+	},
+	"browserslist": [
+		"last 1 Chrome version",
+		"last 1 Firefox version"
+	],
+	"xo": {
+		"envs": [
+			"browser",
+			"webextensions"
+		]
+	},
+	"stylelint": {
+		"extends": "stylelint-config-xo"
+	},
+	"dependencies": {
+		"webext-base-css": "^1.4.1",
+		"webext-options-sync": "^3.1.0"
+	},
+	"devDependencies": {
+		"@parcel/config-webextension": "^2.6.2",
+		"npm-run-all": "^4.1.5",
+		"parcel": "^2.6.2",
+		"stylelint": "^14.9.1",
+		"stylelint-config-xo": "^0.21.0",
+		"web-ext": "^7.2.0",
+		"xo": "^0.50.0"
+	},
+	"webExt": {
+		"sourceDir": "distribution",
+		"run": {
+			"startUrl": [
+				"https://github.com/fregante/browser-extension-template"
```

Comment:
leftover from template or actually useful config?

Reply:
Leftover from a template. No longer works because of the changes I made to support multiple manifest files. Removed it now.